compiler/src/iree/compiler/Codegen/TransformDialectStrategies/GPU/MatmulTensorCoreStrategy.cpp

```diff
@@ -227,7 +227,9 @@ void iree_compiler::gpu::buildMatmulTensorCoreStrategy(
 
   // Step 2. Pad the matmul op.
   auto paddedMatmulOpH =
-      buildPadMatmul(b, tileReductionResult.tiledOpH, strategy);
+      buildPad(b, tileReductionResult.tiledOpH,
+               b.getF32ArrayAttr(strategy.paddingValues),
```

Comment:
Can we add a TODO for constructing the padding values based on the captured types rather than always using F32?